Cargo.toml

```diff
@@ -86,29 +83,30 @@ toml = { version = "0.9.6", default-features = false, features = [
 ] }
 
 # Logging crates
-tracing = "0.1.41"
-tracing-subscriber = { version = "0.3.19", features = ["env-filter"] }
-ansi_term = "0.12.1"
+tracing = { version = "0.1.41", default-features = false, features = ["std"] }
+tracing-subscriber = { version = "0.3.19", default-features = false, features = ["std", "ansi", "fmt", "env-filter"] }
+ansi_term = { version = "0.12.1", default-features = false }
 
 # Web dependencies
 
 ## Opening the app in the browser
-webbrowser = { version = "1.0.2", optional = true }
+webbrowser = { version = "1.0.2", default-features = false, optional = true }
 
 ## Serving the app for the browser
-http = { version = "1.2", optional = true }
+http = { version = "1.2", default-features = false, features = ["std"], optional = true }
 axum = { version = "0.8.5", default-features = false, features = [
     "ws",
     "http1",
     "tokio",
     "tracing",
 ], optional = true }
-tower = { version = "0.5.2", features = ["util"], optional = true }
-tower-http = { version = "0.6.1", features = ["fs", "trace"], optional = true }
-tokio = { version = "1.0", features = ["full"], optional = true }
+tower = { version = "0.5.2", default-features = false, features = ["util"], optional = true }
+tower-http = { version = "0.6.1", default-features = false, features = ["fs", "trace"], optional = true }
+# TODO: Don't enable full features
```

Comment:
Does this TODO still apply?

Reply:
Oops, that's a left over. Will fix it tomorrow